CONTRIBUTING.md

```diff
@@ -0,0 +1,148 @@
+# Contributing Guidelines
+
+### General
+* Keep in mind that this is a design project. Contributions of all kinds (issues, ideas, proposals), not just code, are highly appreciated.
+* Pull requests are welcome with the understanding that changes proposed to actual operations, parameters and responses will be carefully evaluated and discussed, and may not always be accepted.
```

Comment:
These caveats apply to the design of our actual API (we are extra careful there, for many reasons). I'm not sure we need the same kinds of caveats here.